gulp/tasks/styles.js

```diff
@@ -2,22 +2,25 @@ var gulp            = require('gulp'),
     autoprefixer    = require('gulp-autoprefixer'),
     browserSync     = require('browser-sync'),
     changed         = require('gulp-changed'),
-    config          = require('../config').styles,
+    config          = require('../config'),
     handleErrors    = require('../util/handleErrors'),
     minifyCSS       = require('gulp-minify-css'),
     sass            = require('gulp-sass'),
     sourcemaps      = require('gulp-sourcemaps');
 
 gulp.task('styles', function () {
-    return gulp.src(config.pldoc_src_files)
+    return gulp.src([
+            config.lib.src + '/afontgarde/afontgarde.css',
+            config.styles.pldoc_src_files
+        ])
         .pipe(sourcemaps.init())
-        .pipe(changed(config.dest)) // ignore unchanged files
-        .pipe(sass(config.settings_develepment))
+        .pipe(changed(config.styles.dest)) // ignore unchanged files
```

Comment:
Also, added the config specific object over and over here is redundant and less elegant than the way it was.